### DISCUSSION TITLE
Forking policy

### DISCUSSION THREAD

**Discussion by slawomirbabicz** [vectordotdev/vector#16796] [Q&A] [answered]
Hello vector team!
We are trying to fork the repository to our organization and we get the error:

```
You cannot fork this repository to the selected destination due to a policy.
```

Do you have some specific restrictions related to forking this repository?
Thanks

**Comment by spencergilbert** [comment, marked as answer]
I believe that's a policy set on the organization the repo is being forked _to_? I have a fork myself at [spencergilbert/vector](https://github.com/spencergilbert/vector) and I've seen a few other contributions coming from forks.
